src/govuk/components/pagination/template.test.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+const axe = require('../../../../lib/axe-helper')
+
+const { render, getExamples } = require('../../../../lib/jest-helpers')
+
+const examples = getExamples('pagination')
+
+describe('Pagination', () => {
+  describe('Numbered view', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('pagination', examples.default)
+
+      const results = await axe($.html())
+      expect(results).toHaveNoViolations()
+    })
+
+    it('renders the correct URLs for each link', () => {
+      const $ = render('pagination', examples.default)
+      const previous = $('.govuk-pagination__prev .govuk-pagination__link')
```

Comment:
Generally when a variable references a Cheerio HTML element we prefix it with `$` in our tests, so this would be `$previous` for example.